### PR TITLE
Registry functional/integration tests

### PR DESCRIPTION
**Pull request by edigaryev**
Resolves https://github.com/cirruslabs/tart/issues/34.

### CODE REVIEW

File: Tests/TartTests/RegistryTests.swift [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+final class RegistryTests: XCTestCase {
+    var registryRunner: RegistryRunner?
+
+    override func setUp() async throws {
+        try await super.setUp()
+
+        do {
+            registryRunner = try await RegistryRunner()
+        } catch {
+            try XCTSkipIf(ProcessInfo.processInfo.environment["CI"] == nil)
```
Comment (fkorotkov) [inline Tests/TartTests/RegistryTests.swift:13]: Nice!

File: Sources/tart/OCI/Manifest.swift [hunk 2]
```diff
@@ -3,26 +3,26 @@ import Foundation
 let ociManifestMediaType = "application/vnd.oci.image.manifest.v1+json"
 let ociConfigMediaType = "application/vnd.oci.image.config.v1+json"
 
-struct OCIManifest: Encodable, Decodable {
+struct OCIManifest: Encodable, Decodable, Equatable {
```
Comment (fkorotkov) [inline Sources/tart/OCI/Manifest.swift:6]: Let's replace Encodable, Decodable with just `Codable`

**Review by fkorotkov — COMMENTED** [review]
Let's replace Encodable, Decodable with just `Codable`

File: Sources/tart/OCI/Manifest.swift (hunk 2 above)
Comment (edigaryev) [reply Sources/tart/OCI/Manifest.swift]: See https://github.com/cirruslabs/tart/pull/96/commits/694047a07f06f2a4f08e2cb9b9ad17a253d7741b.

**Review by fkorotkov — APPROVED** [review]

**Pull request merged**